### PR TITLE
Fix for windows policy-pack install

### PR DESCRIPTION
**Pull request by jkisk**
Fix filepath so node_modules directory will be found during windows install.  Addresses https://github.com/pulumi/pulumi-policy/issues/171

### CODE REVIEW

File: pkg/npm/npm_test.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	    }
+	}`)
+	assert.NoError(t, ioutil.WriteFile(packageJSONFilename, packageJSON, 0644))
+
+	// Install dependencies, passing nil for stdout and stderr, which connects
+	// them to the file descriptor for the null device (os.DevNull).
+	bin, err := Install(pkgdir, nil, nil)
+	assert.NoError(t, err)
+	assert.Equal(t, expectedBin, bin)
+}
```
Comment (justinvp) [inline pkg/npm/npm_test.go]: Nit: Add a trailing new line.

**Comment by jkisk** [comment]
I added a test kindly provided by justinvp

**Comment by ekrengel** [comment]
We have integration tests... would that not have caught this? 🤔

**Comment by ekrengel** [comment]
Ah the tests don’t test the local policy pack flag. May be worth adding some more tests here: tests/integration/policy/policy_test.go

**Pull request merged**